website/docs/theme-classic.md

```diff
@@ -57,6 +57,7 @@ module.exports = {
       logo: {
         alt: 'Site Logo',
         src: 'img/logo.svg',
+        link: 'https://v2.docusaurus.io/', // default to siteConfig.baseUrl
```

Comment:
Should we call this `href`? More aligned with the `alt` and `src`, which are HTML attributes.

Reply:
Sounds good, fixed!